app/util/Transcoder.scala

```diff
@@ -0,0 +1,28 @@
+package util
+
+import com.amazonaws.services.elastictranscoder.model.ListJobsByPipelineRequest
+import model.transcoder.JobStatus
+import play.api.cache.CacheApi
+
+import scala.collection.JavaConverters._
+import scala.concurrent.duration._
+
+class Transcoder(awsConfig: AWSConfig, cache: CacheApi) {
+
+  def getJobsStatus = cache.getOrElse[List[JobStatus]]("transcoderJobs", 20.seconds){updateJobsStatus}
+
+  private def updateJobsStatus = {
```

Comment:
I'd probably give this function brackets in the definition since it does a whole request to a back-end. Not a dealbreaker